### DISCUSSION TITLE
Branch size

### DISCUSSION THREAD

**Discussion by wipfli** [maplibre/maplibre-native#2062] [Q&A] [answered]
I just pushed a branch which is basically a copy of this repo's main branch and it was more than 50 MB of data. 

Do you know where does the bulk of this come from?

```
➜  maplibre-native git:(fix-tier) ✗ git push --set-upstream wipfli fix-tier         
...
Writing objects: 100% (16854/16854), 51.65 MiB | 1.52 MiB/s, done.
...
```

**Comment by ntadej** [comment]
I suspect that Metal migration introduced new reference files. When did you last time push to your fork?

**Reply by wipfli** [reply]
1 hour ago I pushed a branch based on current main

**Reply by ntadej** [reply]
Yes, but I meant before that, when it was the last time 😊

**Reply by wipfli** [reply]
Huff I don't know. So it is just differential usually?

**Reply by ntadej** [reply, marked as answer]
Yes, it will upload everything on the branch to your fork that you do not have yet. If you make a new branch now it should be almost nothing.

**Reply by louwers** [reply]
Yup sounds like you pushed a lot of commits to your fork that it didn't have yet.

**Reply by wipfli** [reply]
How big is the raw repo?

**Reply by louwers** [reply]
~273 MB

https://github.com/settings/repositories
